src/main/java/com/napier/sem/App.java

```diff
@@ -112,7 +112,7 @@ public void connect()
                 // Wait a bit for db to start
                 Thread.sleep(5000);
                 // Connect to database
-                connection = DriverManager.getConnection("jdbc:mysql://database:3306/world?useSSL=false",
+                connection = DriverManager.getConnection("jdbc:mysql://" + location + "/employees?allowPublicKeyRetrieval=true&useSSL=false",
```

Comment:
i don't think the `/world?` should have changed

Reply:
this caused an error we spent ages looking for. Thanks

Reply:
changed to 
 connection = DriverManager.getConnection("jdbc:mysql://" + location + "/world?allowPublicKeyRetrieval=true&useSSL=false",